src/modules/FavoritesPage/FavoritesPage.module.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../styles/utils/variables/colors';
+@import '../../styles/utils/mixins/mixin-media';
+@import '../../styles/blocks/grid';
+
+.favouritesPage {
```

Comment:
This will have to be a separate independent component that will be reused, so you need to name the classes accordingly. Do not be tied to the component in which you are currently doing it